public/setup/atomicupdate/pathToAccount.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`page`) VALUES ('Planning Hebdo - Mon Compte','100','/myaccount');";
```

Comment:
Provide permissions.yaml
You should delete this entry:

+----+-----------------------------+-----------+--------+---------------+-------+-----------+
| id | nom                         | groupe_id | groupe | page          | ordre | categorie |
+----+-----------------------------+-----------+--------+---------------+-------+-----------+
| 61 | Planning Hebdo - Mon Compte |       100 |        | monCompte.php |     0 |           |
+----+-----------------------------+-----------+--------+---------------+-------+-----------+